Documentation/ceph-dashboard-monitoring.md

```diff
@@ -0,0 +1,329 @@
+- A PV with available space in order to store Grafana database (500Mb)
+- The files with the different resources definition and configuration. You can get them from the Rook repository:
+```console
+git clone --single-branch --branch {{ branchName }} https://github.com/rook/rook.git
+cd rook/cluster/examples/kubernetes/ceph/monitoring
+```
+## Prometheus Operator
+Follow the instructions provided in [Prometheus Monitoring](https://github.com/rook/rook/blob/{{ branchName }}/Documentation/ceph-monitoring.md)
+
+The Prometheus operator allows you to set up a Prometheus instance in you kubernetes cluster, a summary of the commands you will need to execute is below:
+
+Install the Prometheus Operator
+```console
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/v0.40.0/bundle.yaml
+```
+Go to the folder where the different resource files for the monitoring stack are stored
+```console
+cd rook/cluster/examples/kubernetes/ceph/monitoring
+```
+Create and configure the Prometheus instance 
+```console
+kubectl create -f service-monitor.yaml
+kubectl create -f prometheus.yaml
+kubectl create -f prometheus-service.yaml
+```
+
+With these steps you must have a running instance of Prometheus configured to scrape data from the Ceph cluster through the Ceph Prometheus manager module
+
+To verify that prometheus is running you can try to access the prometheus instance.
+
+Execute:
+```console
+echo "http://$(kubectl -n rook-ceph -o jsonpath={.status.hostIP} get pod prometheus-rook-prometheus-0):30900"
+```
+>```
+>http://192.168.122.70:30900
+>```
+If you use the output URL in your browser, you should see the Prometheus monitoring website.
+
+![Prometheus Monitoring Website](media/prometheus-monitor.png)
+
+## Prometheus Ceph manager module
+The source of the Ceph cluster metrics is this manager module, by default is enabled, and the Prometheus operator service monitor configures Prometheus to scrape data from it.
+
+In order to avoid monitoring issues just verify that the scrape intervals are the same in Prometheus and in the Ceph manager module. Both intervals must be the same, depending on your monitoring needs, a value between 5 and 30 is correct.
+
+In the Prometheus web UI, go to Status-Configuration, the scrape configuration should be like:
+
+```yaml
+scrape_configs:
+- job_name: rook-ceph/rook-ceph-mgr/0
+  honor_timestamps: true
+  scrape_interval: 5s
+```
+
+To verify this value in the Ceph Manager module, open a [Rook toolbox](ceph-toolbox.md) and execute:
+```console
+ceph config get mgr mgr/prometheus/scrape_interval
+```
+>```
+>30.000000
+>```
+
+If you have a different value (as you can see in this example), you can change it executing:
+```console
+ceph config set mgr mgr/prometheus/scrape_interval 5
+```
+
+## Grafana
+
+The Grafana image used is a customized image for Ceph. It is Grafana V.6.7.4, and it includes the set of Grafana dashboards used in the Ceph dashboard, and a couple of visualization plugins.
```

Comment:
Only this exact version? Or we need at least v6.7.4?

Reply:
Yes, only the V6.7.4. Even it is more restrictive, because you cannot use a V6.7.4 standard version of the Grafana image, you need to use the Grafana Ceph customized image.
Is not clear in the text provided?